datalad/config.py

```diff
@@ -129,7 +129,7 @@ def _get_default_file_candidates(self):
             cfg_file_candidates.append(opj(home_cfg_base_path, 'datalad.cfg'))
 
         # current dir config
-        cfg_file_candidates.append(opj('.datalad', 'config'))
+        cfg_file_candidates.append('datalad.cfg')
```

Comment:
as talked elsewhere, let's keep `.datalad/config` and better fix the docs, than the code